pkg/generators/servers.go

```diff
@@ -413,14 +413,7 @@ func (g *ServersGenerator) generateServerAdapterSource(resource *concepts.Resour
 }
 
 func (g *ServersGenerator) generateRequestSource(method *concepts.Method) {
-	g.buffer.Import("bytes", "")
-	g.buffer.Import("context", "")
 	g.buffer.Import("encoding/json", "")
-	g.buffer.Import("fmt", "")
-	g.buffer.Import("io/ioutil", "")
-	g.buffer.Import("net/http", "")
-	g.buffer.Import(path.Join(g.base, g.errorsPkg()), "")
-	g.buffer.Import(path.Join(g.base, g.helpersPkg()), "")
```

Comment:
@jhernand the `helpers` pkg is used in the generated code (it is where all the `Parse*` methods are that use for the query parameters).
Either move that code here or keep this import please.

Reply:
Thanks @nimrodshn. I moved the imports to the functions where they are used.